docker/images/dev/Dockerfile

```diff
@@ -16,6 +16,11 @@ RUN apt-get update && \
     vim && \
     rm -rf /var/lib/apt/lists/*
 
+# Install the Crowdin CLI that helps us upload/download strings & translations
+RUN wget -qO - https://artifacts.crowdin.com/repo/GPG-KEY-crowdin | apt-key add - && \
+    echo "deb https://artifacts.crowdin.com/repo/deb/ /" > /etc/apt/sources.list.d/crowdin.list && \
+    apt-get install crowdin
```

Comment:
You should update package list prior to installation, _e.g_ `apt-get update && apt-get install crowdin`

Reply:
Also clear apt cache to decrease image size by adding: `rm -rf /var/lib/apt/lists/*` as the final instruction of this layer (`RUN`).

Reply:
:thinking: After-thought: maybe we can improve the two RUN statements using apt as follow:

```Docker
# Add Crowdin CLI (upload/download strings & translations) apt repository
RUN wget -qO - https://artifacts.crowdin.com/repo/GPG-KEY-crowdin | apt-key add - && \
    echo "deb https://artifacts.crowdin.com/repo/deb/ /" > /etc/apt/sources.list.d/crowdin.list 

# Install development dependencies via apt
RUN apt-get update && \
    apt-get install -y \
      crowdin vim && \
    rm -rf /var/lib/apt/lists/*
```